src/main/java/com/google/sps/travelbud/EventServlet.java

```diff
@@ -0,0 +1,95 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.travelbud;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+// hard coded data
+class Event {
+  int ID;
+  String name;
+  int City_ID;
```

Comment:
[Style](https://google.github.io/styleguide/javaguide.html#s5.2.5-non-constant-field-names): Non-constant field names (static or otherwise) are written in _lowerCamelCase_.

```suggestion
  int id;
  String name;
  int cityId;
```